### PR TITLE
Add CDO support and automatic test launching

### PR DESCRIPTION
**Pull request by arnauddieumegard**
This PR includes and completes the content provided in https://github.com/eclipse/capella-basic-vp/pull/57

### CODE REVIEW

File: perfoviewpoint/plugins/org.polarsys.capella.vp.perfo.design/src/org/polarsys/capella/vp/perfo/design/service/perfoListener/PerfoSessionListener.java [hunk 1]
```diff
@@ -169,13 +167,13 @@ private PhysicalArchitecture retrievesPhysicalArchitectureFromSession(Session se
 		List<PhysicalArchitecture> physicalAchitecture = new ArrayList<>();
 
 		resources.stream().filter(resource -> resource instanceof CapellamodellerResourceImpl).forEach((resource) -> {
```
Comment (pdulth) [inline perfoviewpoint/plugins/org.polarsys.capella.vp.perfo.design/src/org/polarsys/capella/vp/perfo/design/service/perfoListener/PerfoSessionListener.java]: Not sure its working on Team. Its not CDOReosurce?

File: massviewpoint/plugins/org.polarsys.capella.vp.mass.doc/html/Mass/Introduction.html [hunk 2]
```diff
@@ -0,0 +1,17 @@
+<?xml version='1.0' encoding='utf-8' ?><!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml">
+	<head>
+		<meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
```
Comment (pdulth) [inline massviewpoint/plugins/org.polarsys.capella.vp.mass.doc/html/Mass/Introduction.html:4]: Can you generate docs with https://github.com/eclipse/capella/blob/master/doc/plugins/org.polarsys.capella.doc/build.xml to add the css

Ex:
https://github.com/eclipse/capella-sss-transition/blob/master/plugins/org.polarsys.capella.transition.system2subsystem.doc/html/Installation%20and%20Uninstallation.html#L6

File: perfoviewpoint/plugins/org.polarsys.capella.vp.perfo.doc/build.xml [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+<project name="project" default="generate-html">
+	
+	<!-- This property should be set to the path to wikitext standalone package or it can be run inside the current eclipse 
+	 platform if it contains mylyn/mediawiki plugins.
+        
+     Set to ${eclipse.home}/plugins and run the build.xml with the following option checked 
+	 RunAs/ExternalToolsConfiguration/JRE/RunInTheSameJREThanWorkspace
+	-->
+	<property name="wikitext.standalone" value="C:\vrac\capella-studio-5.2.0\plugins"/>
```
Comment (pdulth) [inline perfoviewpoint/plugins/org.polarsys.capella.vp.perfo.doc/build.xml]: https://github.com/eclipse/capella/blob/master/doc/plugins/org.polarsys.capella.doc/build.xml#L23

File: releng/org.polarsys.capella.basic.mass.viewpoint.site/category.xml [hunk 4]
```diff
@@ -14,11 +14,16 @@
...
    <feature id="org.polarsys.capella.vp.mass.feature.source">
       <category name="org.polarsys.capella.basic.mass.viewpoint"/>
    </feature>
+   <feature id="org.polarsys.capella.vp.mass.cdo.feature.source">
+      <category name="org.polarsys.capella.basic.mass.viewpoint.collaboration"/>
+   </feature>
+   <feature id="org.polarsys.capella.vp.mass.tests.feature.source">
+      <category name="org.polarsys.capella.basic.mass.viewpoint.tests"/>
+   </feature>
    <category-def name="org.polarsys.capella.basic.mass.viewpoint" label="Capella Basic Mass Viewpoint"/>
```
Comment (pdulth) [inline releng/org.polarsys.capella.basic.mass.viewpoint.site/category.xml]: Can you remove 'Capella' in the labels of categories.xml to "Basic xx yyy"

File: massviewpoint/features/org.polarsys.capella.vp.mass.feature/feature.xml [hunk 5]
```diff
@@ -68,4 +68,11 @@
          version="0.0.0"
          unpack="false"/>
```
Comment (pdulth) [inline massviewpoint/features/org.polarsys.capella.vp.mass.feature/feature.xml:70]: can you do global pass to EPL-2 rather than EPL-1

**Review by pdulth — CHANGES_REQUESTED** [review]

File: perfoviewpoint/plugins/org.polarsys.capella.vp.perfo.design/src/org/polarsys/capella/vp/perfo/design/service/perfoListener/PerfoSessionListener.java (hunk 1 above)
Comment (pdulth) [reply perfoviewpoint/plugins/org.polarsys.capella.vp.perfo.design/src/org/polarsys/capella/vp/perfo/design/service/perfoListener/PerfoSessionListener.java]: CapellaResourceHelper.isCapellaResource

Comment (arnauddieumegard) [reply perfoviewpoint/plugins/org.polarsys.capella.vp.perfo.design/src/org/polarsys/capella/vp/perfo/design/service/perfoListener/PerfoSessionListener.java]: Thanks for the pointer

File: massviewpoint/plugins/org.polarsys.capella.vp.mass.doc/html/Mass/Introduction.html (hunk 2 above)
Comment (arnauddieumegard) [reply massviewpoint/plugins/org.polarsys.capella.vp.mass.doc/html/Mass/Introduction.html:4]: Done

File: perfoviewpoint/plugins/org.polarsys.capella.vp.perfo.doc/build.xml (hunk 3 above)
Comment (arnauddieumegard) [reply perfoviewpoint/plugins/org.polarsys.capella.vp.perfo.doc/build.xml]: Done

File: releng/org.polarsys.capella.basic.mass.viewpoint.site/category.xml (hunk 4 above)
Comment (arnauddieumegard) [reply releng/org.polarsys.capella.basic.mass.viewpoint.site/category.xml]: Done

File: massviewpoint/features/org.polarsys.capella.vp.mass.feature/feature.xml (hunk 5 above)
Comment (arnauddieumegard) [reply massviewpoint/features/org.polarsys.capella.vp.mass.feature/feature.xml:70]: Done (with a lot of files impacted)

**Comment by pdulth** [comment]
Some Css was refering to org.polarsys.capella.doc/html/styles.css rather than PLUGINS_ROOT/org.polarsys.capella.doc/html/styles.css. I repushed.

**Review by pdulth — APPROVED** [review]

**Pull request merged**